coral/src/domain/topic/topic-api.ts

```diff
@@ -271,7 +271,7 @@ const deleteTopicRequest = ({
 };
 
 const deleteTopic = (params: DeleteTopicPayload) => {
-  // DeleteTopicPayload represents the TopicDeleteRequestModel
+  // KlawApiModel<"TopicDeleteRequestModel"> represents the TopicDeleteRequestModel
```

Comment:
It should be: 

`// DeleteTopicPayload represents the KlawApiModel<"TopicDeleteRequestModel"> without remark (...)`

I think?

Reply:
But I  believe `DeleteTopicPayload` *has* `remark` :o It is the temporary type until `remark` is added to the API.

![Screenshot 2023-07-19 at 15 29 37](https://github.com/aiven/klaw/assets/20607294/4c438186-754b-4ccc-9984-4c5f4b5ca6d5)

Reply:
I think this is a legibility / writing issue haha!

Reply:
AH yes absolutely right! Then it would rather be: `DeleteTopicPayload represents the KlawApiModel<"TopicDeleteRequestModel"> with remark`?

Reply:
[ef412fa](https://github.com/aiven/klaw/pull/1488/commits/ef412fa04f36a1ac305ad8ee1ad01599b3d48d5f)

Reply:
Oh, sorry, I was 💯  sure I did merge it already yesterday 🙈